ui/apps/platform/src/Components/CompoundSearchFilter/attributes/imageCVE.ts

```diff
@@ -23,4 +23,51 @@ export const CVSS: CompoundSearchFilterAttribute = {
     inputType: 'condition-number',
 };
 
-export const imageCVEAttributes = [Name, DiscoveredTime, CVSS];
+export const EPSSProbability: ConditionTextFilterAttribute = {
+    displayName: 'EPSS probability',
+    filterChipLabel: 'EPSS probability',
+    searchTerm: 'EPSS Probability',
+    inputType: 'condition-text',
+    featureFlagDependency: ['ROX_SCANNER_V4', 'ROX_EPSS_SCORE'],
+    inputProps: {
+        conditionProps: {
+            conditionEntries: [
+                ['>', 'Is greater than'],
+                ['>=', 'Is greater than or equal to'],
+                ['=', 'Is equal to'],
+                ['<=', 'Is less than or equal to'],
+                ['<', 'Is less than'],
+            ],
+        },
+        textProps: {
```

Comment:
Although each function is clear and readable, these might benefit a bit from unit tests. Potentially if changes are needed in the future as regex takes a bit to visually parse and could be susceptible to added bugs.

Reply:
Yes, indeed. Given the attributes folder, here is my initial thought:
* Add epssProbability.ts file for functions
* Add epssProbability.test.ts file for unit tests